src/Jaeger/Span.php

```diff
@@ -396,15 +395,15 @@ public function getLogs(): array
     /**
      * {@inheritdoc}
      */
-    public function addBaggageItem($key, $value)
+    public function addBaggageItem(string $key, string $value) :void
     {
         $this->context = $this->context->withBaggageItem($key, $value);
     }
 
     /**
      * {@inheritdoc}
      */
-    public function getBaggageItem($key)
```

Comment:
Travis CI says that there is errors like:
```
PHP Fatal error: 
Declaration of Jaeger\SpanContext::getBaggageItem(string $key): ?string
must be compatible with OpenTracing\SpanContext::getBaggageItem($key)
in /home/travis/build/jonahgeorge/jaeger-client-php/src/Jaeger/SpanContext.php on line 8
```

Reply:
`
 public function getBaggageItem(string $key): ?string;
`
is right  
[SpanContext.php](https://github.com/opentracing/opentracing-php/blob/master/src/OpenTracing/SpanContext.php)

It seems that Travis CI chose opentracing  of  1.0.0-beta5 ?

Reply:
I can't approve/merge this until tests are failed. So, could you please fix you patch?